src/Lean/Meta/UnificationHint.lean

```diff
@@ -26,8 +26,10 @@ structure UnificationHints where
 instance : ToFormat UnificationHints where
   format h := format h.discrTree
 
+def UnificationHints.config : WhnfCoreConfig := {}
```

Comment:
Is this is right value? Unification hints were previously using `simpleReduce := true`, so I thought we should have `iota := ff` here.

Reply:
Good catch. I fill fix it.